tests/cnf/core/network/sriov/tests/rdmametrics.go

```diff
@@ -133,7 +133,7 @@ var _ = Describe("rdmaMetrics", Ordered, Label(tsparams.LabelRdmaMetricsTestCase
 				Expect(err).ToNot(HaveOccurred(), "Failed to delete rdma exclusive mode machine config")
 
 				err = netenv.WaitForMcpStable(
-					APIClient, tsparams.MCOWaitTimeout, tsparams.DefaultStableDuration, NetConfig.CnfMcpLabel)
+					APIClient, tsparams.MCOWaitTimeout, 1*time.Minute, NetConfig.CnfMcpLabel)
```

Comment:
You have another 1*time.Minute on line 71. Maybe make a var?

Reply:
Can i change below var to 1 minute or else add a new var ?

https://github.com/openshift-kni/eco-gotests/blob/44e389f89adf048805cfbccfb124e3b2d7c148cc/tests/cnf/core/network/sriov/internal/tsparams/sriovvars.go#L21